devices/datek.js

```diff
@@ -85,13 +85,32 @@ module.exports = [
             e.voltage_phase_c(), e.temperature()],
     },
     {
-        fingerprint: [{modelID: 'Motion Sensor', manufacturerName: 'Eva'}],
+        zigbeeModel: ['Motion Sensor'],
```

Comment:
Was this removed on purpose? (preferable this should be kept to prevent conflicts with devices using the same modelID)

Reply:
Yes it was. No reason to actually remove it, do you want me to put it back ?